transform/operations_test.go

```diff
@@ -62,6 +63,19 @@ func runOpTests(t *testing.T, opType func() transformOperation, tests []opTests)
 		switch {
 		case test.wantErr && err != nil:
 			continue
+		case strings.Contains(test.description,"currentTime") == true:
```

Comment:
no need for == true

Reply:
also this seems sketchy. I think it'd be better if you didn't use runOpTests.

Reply:
missed the == true. 

And to the point - you're cluttering up a common test function a lot with test specific logic. It should get moved out, or somehow otherwise leveraged / modified.